blueflood-core/src/main/java/com/rackspacecloud/blueflood/tools/ops/FixTTL.java

```diff
@@ -0,0 +1,118 @@
+package com.rackspacecloud.blueflood.tools.ops;
+
+import org.apache.commons.cli.*;
+import com.netflix.astyanax.model.*;
+import com.rackspacecloud.blueflood.io.*;
+import com.rackspacecloud.blueflood.types.*;
+import java.util.*;
+
+public class FixTTL {
+    // Update all the TTL's for a set of metrics over a particular time range
+    public static void fixTTLs(ColumnFamily CF, String tenantID,
+                               List<String> metrics, Range range, Integer newTTL)
+            throws Exception {
+
+        List<Locator> locators = new ArrayList<Locator>();
+        for (String path: metrics) {
+            locators.add(Locator.createLocatorFromPathComponents(tenantID, path));
+        }
+
+        // Print out current TTL's, and update them
+        Map<Locator, ColumnList<Long>> data =
+                AstyanaxReader.getInstance().getColumnsFromDB(locators, CF, range);
+
+        for (Map.Entry<Locator, ColumnList<Long>> entry: data.entrySet()) {
+            Locator l = entry.getKey();
+            System.out.println("Current Locator is " + l.toString());
+            ColumnList<Long> cols = entry.getValue();
+
+            System.out.println("Number of cols is " + cols.size());
+            for (Column<Long> col: cols) {
+                System.out.println("col name is " + col.getName() + " ttl is " + col.getTtl());
```

Comment:
What do you think about having a `--verbose` option for printing the name of every col, to avoid printing too much data, unless user need it?